src/baitapquanlyphuongtiengiaothong/QuanLyPTGT.java

```diff
@@ -0,0 +1,108 @@
+package baitapquanlyphuongtiengiaothong;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Scanner;
+
+public class QuanLyPTGT {
+	int soLuongOto = 0;
+	int soLuongXeMay = 0;
+	int soLuongXeTai = 0;
+	private ArrayList<PhuongTienGiaoThong> ds = new ArrayList<PhuongTienGiaoThong>();
+
+	public void nhapPhuongTien(Scanner sc) {
+		System.out.print("Nhập loại phương tiện giao thông: ");
+		String loaiXe = sc.nextLine();
+		switch (loaiXe) {
+		case "OTo":
+			OTo oTo = new OTo();
+			oTo.nhap(sc);
+			ds.add(oTo);
+			soLuongOto++;
+			break;
+		case "XeMay":
+			XeMay xeMay = new XeMay();
+			xeMay.nhap(sc);
+			ds.add(xeMay);
+			soLuongXeMay++;
+			break;
+		case "XeTai":
+			XeTai xeTai = new XeTai();
+			xeTai.nhap(sc);
+			ds.add(xeTai);
+			soLuongXeTai++;
+			break;
+		default:
+			System.out.println("Không có loại xe bạn cần nhập!");
+			System.out.println("Vui lòng nhập lại: ");
+			nhapPhuongTien(sc);
+		}
```

Comment:
```suggestion
		System.out.print("Nhập loại phương tiện giao thông: ");
		String loaiXe = sc.nextLine();
		Xe newXe;
		switch (loaiXe) {
		case "OTo":
			newXe = new OTo();
			soLuongOto++;
			break;
		case "XeMay":
			newXe = new XeMay();
			soLuongXeMay++;
			break;
		case "XeTai":
			newXe = new XeTai();
			soLuongXeTai++;
			break;
		default:
			System.out.println("Không có loại xe bạn cần nhập!");
			System.out.println("Vui lòng nhập lại: ");
			nhapPhuongTien(sc);
		}
		newXe.nhap(sc);
		ds.add(newXe);
```
Như ri sẽ đỡ lặp 2 dòng code trong các case đó nhé